src/clients/InventoryClient.ts

```diff
@@ -298,13 +299,17 @@ export class InventoryClient {
   // Return the upcoming refunds (in pending and next bundles) on each chain.
   async getBundleRefunds(l1Token: string): Promise<{ [chainId: string]: BigNumber }> {
     let refundsToConsider: CombinedRefunds[] = [];
-
+    const taskProfiler = new Profiler({
+      logger: this.logger,
+      at: "InventoryClient::getBundleRefunds",
+    });
+    const A = taskProfiler.start("A", {
+      l1Token,
+    });
```

Comment:
Does it work if we instantiate the profiler as a class variable in the constructor, and then change the code here as follows?

```suggestion
    const profiler = this.taskProfiler.start("getBundleRefunds", {
      l1Token,
    });
```